float32 height_request
---
float32 height_result
---
float32 partial_height 